ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/graphql/GraphQLContextType.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright ConsenSys AG.
```

Comment:
I'm sure this was just copied from elsewhere in the code, but we are trying to make sure new files are `Copyright Hyperledger Besu Contributors` moving forward.

Reply:
thank you @jflo , will update!